Tools/LambdaTestTool/src/Amazon.Lambda.TestTool/TestToolStartup.cs

```diff
@@ -318,6 +318,12 @@ private static string DeterminePayload(LocalLambdaOptions localLambdaOptions, Co
                 }
             }
 
+            // Enclose simple string payload in double quotes so that it's parsed properly by Lambda JSON serializer.
+            if (!payload.StartsWith("{") && !payload.StartsWith("\""))
```

Comment:
I'm worried the logic we added in Amazon.Lambda.Tools isn't great logic to duplicate. The problem with the Amazon.Lambda.Tools logic is what if you want to send a bool or an int. This logic automatically converts `true` to `"true"` or `100` to `"100"`.

I wonder instead if we should attempt to parse the JSON using with System.Text.Json. If that parses with no problems then send as is. If it doesn't parse add surrounding quotes if it doesn't already start with quotes then use that value. We could retrofit Amazon.Lambda.Tools with the same logic. I wouldn't view that as a breaking change there and we would fix some scenarios that aren't working correct in Amazon.Lambda.Tools.

Reply:
Implemented fix. Please review.